test/user.test.js

```diff
@@ -1763,6 +1763,54 @@ describe('User', function() {
     });
   });
 
+  describe('password reset with/without email verification', function() {
+    it('allows resetPassword by email if email verification is required and done',
+  function(done) {
+    User.settings.emailVerificationRequired = true;
+    var email = 'foo1@bar.com';
```

Comment:
This is difficult to understand. Please use `validCredentialsEmailVerified.email` instead.